config/rslib-config/index.ts

```
@@ -0,0 +1,59 @@
+import { defineConfig, LibConfig } from '@rslib/core';
+
+type LibFormat = LibConfig['format'];
+export type BundleType = Boolean | 'excludeExternal';
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)